kotlinx-coroutines-core/common/src/CoroutineDispatcher.kt

```diff
@@ -79,6 +79,15 @@ public abstract class CoroutineDispatcher :
      */
     public abstract fun dispatch(context: CoroutineContext, block: Runnable)
 
+    /**
+     * Reserved for [Dispatchers.Unconfined] that returns `false` here because its
+     * [dispatch] method throws [UnsupportedOperationException]. It is used in [yield] implementation.
+     *
+     * @suppress **This an internal API and should not be used from general code.**
+     */
+    @InternalCoroutinesApi
+    public open fun isDispatchSupported(context: CoroutineContext): Boolean = true
```

Comment:
What's the point of `context` parameter?

Reply:
Irrelevant, got rid of the method.